framework/tlvf/src/include/tlvf/WSC/AttrList.h

```diff
@@ -0,0 +1,139 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2019 Tomer Eliyahu (Intel Corporation)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#ifndef _TLVF_WSC_ATTR_LIST_H_
+#define _TLVF_WSC_ATTR_LIST_H_
+
+#include <memory>
+#include <tlvf/ClassList.h>
+#include <tlvf/WSC/WSC_Attributes.h>
+#include <tlvf/ieee_1905_1/tlvWsc.h>
+#include <tlvf/tlvflogging.h>
+
+namespace WSC {
+
+typedef struct sWscAttrHeader {
+    uint16_t type;
+    uint16_t length;
+    uint8_t *data() { return ((uint8_t *)this + sizeof(*this)); }
+} __attribute__((packed)) sWscAttrHeader;
+
+/**
+ * @brief wsc configuration
+ * 
+ */
+struct config {
```

Comment:
`config` is a very generic name... I would consider changing this to a more meaningful name (especially since it's in a header file)

Reply:
It is part of the WSC namespace - so the usage is `WSC::config`, don't you think its enough? it is precisely that - a WSC configuration for creating WSC messages. Do you have a better name to suggest?

Reply:
Depends on the amount of classes in the WSC namespace. If you believe there won't be any other "configuration" structure and/or related data-types it's probably fine.

Reply:
I think its fine, if there will be an issue we can always change it in the future.